src/native/corehost/hostmisc/pal.windows.cpp

```diff
@@ -657,9 +657,31 @@ pal::string_t pal::get_current_os_rid_platform()
     return ridOS;
 }
 
+namespace
+{
+    bool is_directory_separator(pal::char_t c)
+    {
+        return c == DIR_SEPARATOR || c == L'/';
+    }
+}
+
 bool pal::is_path_rooted(const string_t& path)
 {
-    return path.length() >= 2 && path[1] == L':';
+    return (path.length() >= 1 && is_directory_separator(path[0]))
```

Comment:
This seems like it would be false for `C:`. `C` is neither `DIR_SEPARATOR` or `/`. I would assume that `C:\foo` be considered rooted.